components/menu/menu-item-link.js

```diff
@@ -39,7 +39,7 @@ class MenuItemLink extends MenuItemMixin(LitElement) {
 	firstUpdated() {
 		super.firstUpdated();
 		this.addEventListener('keydown', this._onKeyDown);
-		this.shadowRoot.querySelector('a').addEventListener('click', this._onClick);
+		this.addEventListener('click', this._onClick);
```

Comment:
Won't this change cause two click events on the anchor when on desktop?

Reply:
No, so it seems by using a `CustomEvent` for the click, `bubbles` is false and so only one `d2l-menu-item-select` event is dispatched (whereas if I were to do something like `this.shadowRoot.querySelector('a').click()` there are two `d2l-menu-item-select` events dispatched.